### PR TITLE
Docs for arangoinspect

### PR DESCRIPTION
**Pull request by Simran-B**
Initial draft. I'm not sure about the following:

> For non-cluster deployments,
> you may authenticate with a user name and password instead:
> 
>     arangoinspect --server.ask-jwt-secret false --server.username "root" --server.password "foobar"

Is a JWT-secret only used in clusters, or also other deployment modes? Is it only possible to use username/password for single server instances?

### CODE REVIEW

**Comment by Simran-B** [comment]
Builds: https://docs.arangodb.com/review_arangoinspect/Manual/Programs/

File: Documentation/Books/Manual/Programs/Arangoinspect/Examples.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+Arangoinspect Examples
+======================
+
+If you are asked by ArangoDB support to provide an `arangodb-inspector.json`
+file, run the _arangoinspect_ binary to generate in the current working folder.
+
+The resulting JSON file contains the names of existing databases, their
```
Comment (kvahed) [inline Documentation/Books/Manual/Programs/Arangoinspect/Examples.md]: The resulting JSON file is a collection of meta data acquired from all involved instances. The data includes relevant operating system parameters, arangodb process parameters, local database information etc.

File: Documentation/Books/Manual/Programs/Arangoinspect/Examples.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+collections, indexes, shards and some other potentially useful data about
+the system to diagnose problems.
+
+{% hint 'warning' %}
```
Comment (kvahed) [inline Documentation/Books/Manual/Programs/Arangoinspect/Examples.md]: please remove this block. we don't collect data. all collected is meta data. no user names no secrets, no passwords.

Comment (Simran-B) [reply Documentation/Books/Manual/Programs/Arangoinspect/Examples.md]: As far as I've seen, some file paths are included which potentially contain the user name (which may or may not be a real name), also process information (`top`) in which user names are also listed. It's unlikely, but database, collection and index field names could contain something the user doesn't want to share. @sleto-it ?

Comment (sleto-it) [reply Documentation/Books/Manual/Programs/Arangoinspect/Examples.md]: @Simran-B I think the _hint_ you have included about double-checking the generated info and obfuscating some of them, if needed, is a good one

**Comment by sleto-it** [comment]
Hi, are we all set for this PR? @Simran-B if yes, please remove the _WIP_ label and I will merge.

**Comment by Simran-B** [comment]
@sleto-it There's still my open question:

> Is a JWT-secret only used in clusters, or also other deployment modes? Is it only possible to use username/password for single server instances?

And Kaveh asked to remove the hint box about data privacy - we need a decision from you

**Comment by Simran-B** [comment]
Note to self: Also update text if needed in 3.3 regarding JWT secret - https://github.com/arangodb/arangodb/pull/5703

**Pull request merged**